main.go

```diff
@@ -49,16 +50,19 @@ func sendSecret() {
 		log.Fatal(err)
 	}
 
-	fmt.Println("Paste your secret here:")
-	plaintext := readline()
+	fmt.Println("Copy your secret to your clipboard, then press Enter/Return:")
```

Comment:
I love the idea of using the clipboard, but I don't love the idea of _always_ using the clipboard.

How about only performing `pbpaste()` if the `readline()` returns an empty string? For smaller secrets I definitely like to see what I'm about to encrypt and send.

Reply:
That could work. I'll come back to this PR this week with potentially a few other options to move forward with.

Reply:
I found an alternate approach to this, which I'll push up a bit later.